app/store/middleware.js

```diff
@@ -131,7 +131,8 @@ function resetStateForNewVersion(action) {
             thread: {
                 drafts: threadDrafts
             },
-            selectServer
+            selectServer,
+            recentEmojis: payload.views.recentEmojis
```

Comment:
can we please validate that payload.views exists before doing this? I think we did something like that for the other parts of the cleanup

Reply:
Will do